pkg/scheduler/algorithm/predicates/error.go

```diff
@@ -67,8 +67,6 @@ var (
 	ErrNodeNotReady = NewPredicateFailureError("NodeNotReady", "node(s) were not ready")
 	// ErrNodeNetworkUnavailable is used for NodeNetworkUnavailable predicate error.
 	ErrNodeNetworkUnavailable = NewPredicateFailureError("NodeNetworkUnavailable", "node(s) had unavailable network")
-	// ErrNodeUnschedulable is used for NodeUnschedulable predicate error.
-	ErrNodeUnschedulable = NewPredicateFailureError("NodeUnschedulable", "node(s) were unschedulable")
 	// ErrNodeUnknownCondition is used for NodeUnknownCondition predicate error.
 	ErrNodeUnknownCondition = NewPredicateFailureError("NodeUnknownCondition", "node(s) had unknown conditions")
```

Comment:
I didn't remove this because it still used in

```git
pkg/scheduler/algorithm/predicates/predicates.go:               return false, []PredicateFailureReason{ErrNodeUnknownCondition}, nil
``` 

/cc @ahg-g

Reply:
sg